openmetadata-ui/src/main/resources/ui/src/components/common/DatePickerMenu/DatePickerMenu.component.tsx

```diff
@@ -163,10 +162,6 @@ const DatePickerMenu = ({
               <DatePicker.RangePicker
                 bordered={false}
                 clearIcon={<CloseCircleOutlined />}
-                defaultValue={[
```

Comment:
We have removed ability to ser default range, Isn't this cause issue at other place?
Can you add tests for the same?

Reply:
It won't create an issue, as we are showing the date on the label, and storing that label in the state, and in case we pass the default value via props it's being stored in the state and shown as a label. I'll check for the unit test!